README.md

```diff
@@ -1,3 +1,10 @@
+<p>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/logo.svg">
```

Comment:
kinda make sense, just the first time I see this usage!

Reply:
opted to use a bg instead 
![image](https://user-images.githubusercontent.com/34844044/217790924-8d208cd2-2c63-404a-8804-6125a0e2cb20.png)